docs/examples/graph_kernels.py

```diff
@@ -158,6 +174,7 @@
     model=posterior,
     objective=jit(gpx.ConjugateMLL(negative=True)),
     train_data=D,
+    max_iters=1000
```

Comment:
did this fix it? Seems like a huge number for LBFGS

Reply:
No - the error persists. Works locally